entity-framework/core/providers/provider-log.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
Page needs to be added to toc

Reply:
Will do this separately and also link from "writing a provider" page.